draft-ietf-jsonpath-base.md

```diff
@@ -1300,8 +1300,9 @@ the comparison is between values which are both numbers or both strings and whic
       lower Unicode scalar value than the second string or if both strings start with the same Unicode scalar value and
       the remainder of the first string compares less than the remainder of the second string.
 
-Note that comparisons using the operator `<` yield false if either value being
-compared is an object, array, boolean, or `null`.
+Note that comparisons using the operator `<` yield false if either side of the
+comparison results in an empty nodelist or `Nothing` or if either side is a value that
+is an object, array, boolean, or `null`.
```

Comment:
Has the term "Nothing" (especially since it's presented in code font) been introduced at this point?  If not, I think you need at least "(see below)".

Reply:
No, good catch! It was actually used near the start of 2.3.5.2.2, so I added a cross-reference at that point (in 2874edff6356d009917b426d45f73beb033f4613).